mgr/src/server/plugins/node_pool_rename.cr

```diff
@@ -0,0 +1,31 @@
+require "./helpers"
+require "../conf"
+require "./volume_utils.cr"
+
+ACTION_NODE_POOL_RENAME = "node_pool_rename"
+
+node_action ACTION_NODE_POOL_RENAME do |data, _env|
+  puts "data: #{data}, typeof: #{typeof(data)}"
+  data = Hash(String, String).from_json(data)
+  new_pool_name = data["new_pool_name"]
+
+  puts "globalconfig: #{GlobalConfig.local_node}"
+  puts "globalconfig node name: #{GlobalConfig.local_node.name}"
+
+  data_file = Path.new(GlobalConfig.workdir, "info")
+  if !File.exists?(data_file)
+    next NodeResponse.new(false, {"error": "Node info file is not found in #{GlobalConfig.local_node.name}. Restart kadalu-mgr to regenerate the info file"}.to_json)
```

Comment:
if node file not exists then it is a serious issue here. Remove Restart note from the message.

Reply:
Removed last msg.